lib/widgets/payment_request_dialog.dart

```diff
@@ -127,7 +96,45 @@ class PaymentRequestDialogState extends State<PaymentRequestDialog>
         .listen((settings) => _accountSettings = settings);
 
     _sentPaymentResultSubscription = widget.accountBloc.fulfilledPayments.listen((fulfilledPayment) {
-      // Trigger the collapse animation and show flushbar 200ms after the animation is completed
+      controller = AnimationController(
+          vsync: this, duration: Duration(milliseconds: 600));
+      borderAnimation = Tween<double>(begin: 0.0, end: 8.0).animate(
+          CurvedAnimation(parent: controller, curve: Curves.slowMiddle));
+      opacityAnimation = Tween<double>(begin: 0.0, end: 1.0).animate(
+          CurvedAnimation(parent: controller, curve: Curves.slowMiddle));
+      colorAnimation = new ColorTween(
+        begin: theme.BreezColors.blue[500],
+        end: theme.BreezColors.white[500],
+      ).animate(controller)
+        ..addListener(() {
+          setState(() {});
+        });
+
+      RenderBox _paymentTableBox = widget.tableKey.currentContext
+          .findRenderObject();
+      var _paymentItemStartPosition = _paymentTableBox
+          .localToGlobal(Offset.zero)
+          .dy - 32.0;
+      var _paymentItemEndPosition = (MediaQuery
+          .of(context)
+          .size
+          .height - _paymentItemStartPosition) - 72.0 - 32.0;
```

Comment:
I am guessing the 7.2.0 is the item height, there is already a constant defined for this number, we should use it instead.